<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumEditorWidget</class>
 <widget class="QWidget" name="SpectrumEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrum Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Spectrums</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonDeleteSpectrum">
        <property name="toolTip">
         <string>Remove spectrum</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/neon/images/icons/list-remove-icon.png</normaloff>:/neon/images/icons/list-remove-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QListWidget" name="listWidgetSpectrums"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonAddSpectrum">
        <property name="toolTip">
         <string>Add spectrum</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/neon/images/icons/list-add-icon.png</normaloff>:/neon/images/icons/list-add-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxSpectrumProperties">
     <property name="title">
      <string>Spectrum Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxOverwrite">
        <property name="toolTip">
         <string>Overwrite graphics material colour with spectrum colour</string>
        </property>
        <property name="text">
         <string>Overwrite</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxPreview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="SceneviewerWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxComponents">
     <property name="title">
      <string>Components</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonMoveUpSpectrumComponent">
        <property name="toolTip">
         <string>Move component up</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/neon/images/icons/go-up-icon.png</normaloff>:/neon/images/icons/go-up-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="7">
       <widget class="QListWidget" name="listWidgetSpectrumComponents"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonAddSpectrumComponent">
        <property name="toolTip">
         <string>Add component</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/neon/images/icons/list-add-icon.png</normaloff>:/neon/images/icons/list-add-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonDeleteSpectrumComponent">
        <property name="toolTip">
         <string>Remove component</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/neon/images/icons/list-remove-icon.png</normaloff>:/neon/images/icons/list-remove-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5" colspan="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButtonMoveDownSpectrumComponent">
        <property name="toolTip">
         <string>Move component down</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/neon/images/icons/go-down-icon.png</normaloff>:/neon/images/icons/go-down-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxComponentProperties">
     <property name="title">
      <string>Component Properties</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Colour Map:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBoxColourMap"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scale:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxScale"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Index of component to use from data field</string>
        </property>
        <property name="text">
         <string>Data field comp.:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxDataFieldComponent">
        <property name="toolTip">
         <string>Index of component to use from data field</string>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Normalised Range</string>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="toolTip">
            <string>Component to use from data field</string>
           </property>
           <property name="text">
            <string>Min:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditNormalisedRangeMin">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="toolTip">
            <string>Component to use from data field</string>
           </property>
           <property name="text">
            <string>Max:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditNormalisedRangeMax">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Range</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="toolTip">
            <string>Component to use from data field</string>
           </property>
           <property name="text">
            <string>Min:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditRangeMin">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="toolTip">
            <string>Component to use from data field</string>
           </property>
           <property name="text">
            <string>Max:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditRangeMax">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonAutorange">
        <property name="text">
         <string>Autorange</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxScene"/>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="checkBoxReverse">
        <property name="text">
         <string>Reverse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneviewerWidget</class>
   <extends>QWidget</extends>
   <header>opencmiss/neon/ui/zincwidgets/sceneviewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
